cloudbuild/fp-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+
+git clone https://github.com/openconfig/featureprofiles.git
+cd ... || exit
+
+function metadata_kne_topology() {
```

Comment:
this can be simplified to always use the openconfig/lemming/topology.textproto as the resources differences between topos is negligible (not that this approach is wrong)

Reply:
thanks, done